packages/spezi-firebase-utils/examples/basic-usage.ts

```diff
@@ -0,0 +1,68 @@
+//
+// This source file is part of the Stanford Biodesign Digital Health Spezi Firebase Remote Notifications open-source project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import { z } from 'zod'
+import {
+  SchemaConverter,
+  LocalizedText,
+  average,
+  chunks,
+  capitalize,
+  advanceDateByDays,
+  optionalish
+} from 'spezi-firebase-utils'
+
+// 1. Define schemas with SchemaConverter
+const userSchema = z.object({
+  name: z.string(),
+  age: z.number().optional(),
```

Comment:
You could also use optionalish and optionalishDefault in this example, in case they have been moved here as well